tst/schedulers/test_schedulers_api.py

```diff
@@ -65,6 +65,7 @@ def make_ray_skopt():
     SimpleScheduler(config_space=config_space, metric=metric1),
     RandomSearch(config_space=config_space, metric=metric1),
     BayesianOptimization(config_space=config_space, metric=metric1),
+    REA(config_space=config_space, metric=metric1, population_size=1, sample_size=2),
```

Comment:
Here, you'd then do:

REA(config_space, metric=metric1, search_options={'population_size': 1, 'sample_size': 2})